lib/perl/Genome/Model/Tools/Joinx/VcfMerge.pm

```diff
@@ -168,6 +173,9 @@ sub _resolve_flags {
     if ($self->merge_samples) {
         $flags .= " -s";
     }
+    if ($self->allow_same_file) {
```

Comment:
I generally agree with you that we should avoid duplicating these checks. However, adding this version check to __planned_errors__ allows the variant reporting framework to fail early at validation time before the process/workflow is even kicked off so I will keep it in the vep expert (and add it to joinx vcf-merge itself).